spec/models/step_by_step_page_spec.rb

```diff
@@ -216,6 +216,27 @@
     end
   end
 
+  describe 'should_check_links?' do
+    it 'should return false if step by step is scheduled' do
+      step_by_step_with_step = create(:step_by_step_page, status: 'scheduled', scheduled_at: 1.day.from_now)
```

Comment:
The `step_by_step_page` could be created once in a `let` block, and the status updated in each test as needed. e.g. here the arrangement could be `step_by_step_page.status == "scheduled"`.

Reply:
Or use the `scheduled_step_by_step_page` factory.